shapes/lv.ttl

```diff
@@ -0,0 +1,143 @@
+    - at least one rml:field property, with a rml:Field as its value.
+    """ ;
+    sh:targetClass rml:LogicalView ;
+    sh:targetObjectsOf rml:parentLogicalView ;
+    sh:property [
+        sh:description """
+        The source of the logical view. This is the logical source from which the logical view is derived.
+        """ ;
+        sh:message """
+        Exactly one rml:source property must be specified for a rml:LogicalView, with a rml:LogicalSource as its value.
+        """ ;
+        sh:path rml:source ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node rmlsh:LogicalSource ;
+        sh:nodeKind sh:BlankNodeOrIRI ;
+    ] , [
+        sh:description """
+        A field of the logical view.
+        """ ;
+        sh:message """
+        At least one rml:field property must be specified for a rml:LogicalView, with a resource as its value.
+        """ ;
+        sh:path rml:field ;
+        sh:minCount 1 ;
+        sh:nodeKind sh:BlankNodeOrIRI ;
+    ] , [
+        sh:description """
+        A left join of the logical view.
+        """ ;
+        sh:message """
+        The value of a rml:leftJoin property for a rml:LogicalView must be a resource.
+        """ ;
+        sh:path rml:leftJoin ;
+        sh:nodeKind sh:BlankNodeOrIRI ;
+    ] , [
+        sh:description """
+        An inner join of the logical view.
+        """ ;
+        sh:message """
+        The value of a rml:innerJoin property for a rml:LogicalView must be a resource.
+        """ ;
+        sh:path rml:innerJoin ;
+        sh:nodeKind sh:BlankNodeOrIRI ;
+    ] .
+
+rmlsh:FieldShape a sh:NodeShape ;
+    sh:name "Field" ;
+    sh:description """
+    A field is an expression map that gives a name to an expression.
+    """ ;
+    sh:message """
+    rml:Field requires a rml:fieldName and may have zero or more rml:field properties.
+    Furthermore, rml:Field requires what an rml:ExpressionMap requires.
+    """ ;
+    sh:targetClass rml:Field ;
+    sh:targetObjectOf rml:field ;
+    sh:property [
+        sh:description """
+        The name of the field.
+        """ ;
+        sh:message """
+        Exactly one rml:fieldName property must be specified for a rml:Field, with a string as its value.
+        """ ;
+        sh:path rml:fieldName ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:datatype xsd:string ;
+        sh:nodeKind sh:Literal ;
+    ] , [
+        sh:description """
+        A child field of the field.
+        """ ;
+        sh:message """
+        The value of a rml:field property for a rml:Field must be a resource.
+        """ ;
+        sh:path rml:field ;
+        sh:nodeKind sh:BlankNodeOrIRI ;
+    ] .
+
+rmlsh:LogicalViewJoinShape a sh:NodeShape ;
+    sh:name "LogicalViewJoin" ;
+    sh:description """
+    A LogicalViewJoin is an operation that extends the logical iteration of one logical view (the child logical view) with fields from another logical view (the parent logical view).
```

Comment:
I like the description and it fits perfectly in case of rml:leftJoin, however, in case of rml:innerJoin the logical iteration of the child logical view can be changed/reduced to less iterations. I am not sure how to capture this.